molecule/default/tests/test_default.py

```diff
@@ -25,7 +25,7 @@ def test_kdc_conf(host):
     assert kdc_conf.is_file
     assert kdc_conf.user == 'root'
     assert kdc_conf.group == 'root'
-    assert oct(kdc_conf.mode) == '0600'
+    assert oct(kdc_conf.mode) == '0o600'
```

Comment:
Actually the proper+shorter way to check file modes (and that works in both ansible v2.7.x and v2.8.x) is 
```
    assert kdc_conf.mode == 0o600
```
I used the same fix in my other PR (that introduces a 2nd test scenario):
https://github.com/ultratendency/kerberos_server/pull/22/commits/16f9945e59dd0ba279b006038ee2717d396759b4

Will now fix also in this PR (so we can still decide to downgrade again to latest released molecule 2.20.1, requiring ansible 2.7x)